projects/detr3d/mmdet3d_plugin/datasets/transforms/transform_3d.py

```diff
@@ -0,0 +1,107 @@
+import mmcv
```

Comment:
Please rename the fold `mmdet3d_plugin` as `detr3d`.

Reply:
projects/detr3d/detr3d/datasets/xxx